Src/Sarif.PartnerMatcher.Function/SpamAnalyzer.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Function
+{
+    public class SpamAnalyzer
+    {
+        public static SarifLog Analyze(string filePath, string text, string rulePath)
+        {
+            string logContents = text;
+
+            string regexDefinitions = Path.Combine(rulePath, @"..\Rules\SEC1001.json");
+
+            var fileSystem = new FileSystem();
```

Comment:
>fileSystem [](start = 16, length = 10)

create the static constructor, use this as static variable to prevent many instances. I think we can use FilSystem.Instance to facilitate

Reply:
done